<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>endpage</class>
 <widget class="QWizardPage" name="endpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Summary:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Microclimate:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Extreme Heat</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="chmExtreme">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../icons.qrc">:/Icons/ressources/checkmark.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Average Summer Heat</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="chmAverage">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../icons.qrc">:/Icons/ressources/checkmark.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Stormwater:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Stream Erosion &amp; Minor Flooding</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="chmSEI">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../icons.qrc">:/Icons/ressources/checkmark.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Treatment &amp; Harvesting Performance</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="chmTreatment">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../icons.qrc">:/Icons/ressources/checkmark.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Stream Hydrology &amp; Water Quality</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="chmHydro">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../icons.qrc">:/Icons/ressources/checkmark.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Future Rainfall</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="chmRain">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../icons.qrc">:/Icons/ressources/checkmark.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
